drizzle-orm/src/sqlite-core/dialect.ts

```diff
@@ -567,7 +567,10 @@ export abstract class SQLiteDialect {
 				} of selectedRelations
 			) {
 				const normalizedRelation = normalizeRelation(schema, tableNamesMap, relation);
-				const relationTableName = relation.referencedTable[Table.Symbol.Name];
+				const relationTableName = relation.referencedTable[Table.Symbol.Schema]
```

Comment:
Given that [SQLite doesn't have support for schemas](https://orm.drizzle.team/docs/schemas), is this needed?
Or maybe, it's just a way to be future-proof and keep the code aligned between the 3 DBs.